### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by KamiyaGaikwad**

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]
good to merge

File: scripts/docs.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+// const hideSearchLinks = (e) => {
+//     if(e.target != searchLinks){
+//     searchLinks.classList.add("hide");}
+//     else{
+//         searchLinks.classList.remove("hide");
+//     }
+// }
+
+// document.addEventListener("click",hideSearchLinks);
+
```
Comment (romabulani) [inline scripts/docs.js]: You should remove these comments if not required

File: scripts/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+// const hideSearchLinks = (e) => {
+//     if(e.target != searchLinks){
+//     searchLinks.classList.add("hide");}
+//     else{
+//         searchLinks.classList.remove("hide");
+//     }
+// }
+
+// document.addEventListener("click",hideSearchLinks);
+
```
Comment (romabulani) [inline scripts/index.js]: Here also

File: styles/docs.css [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+}
+/* 
+.nav-vertical-section{
+    grid-column-start: 3;
+    background-color: var(--dark-orange);
+    position: relative;
+    height: 100%;
+    padding-top:1rem;
+} */
+
```
Comment (romabulani) [inline styles/docs.css]: These comments also

**Review by romabulani — APPROVED** [review]
Rest all looks good to merge.

File: scripts/docs.js (hunk 1 above)
Comment (KamiyaGaikwad) [reply scripts/docs.js]: I am thinking of editing the commented code for future functionalities. That's why I didn't remove it

**Pull request merged**